web/html/src/manager/content-management/shared/components/panels/environment-lifecycle/environment-form.js

```diff
@@ -54,17 +54,12 @@ const EnvironmentForm = (props: Props) =>
             name="predecessorLabel"
             label={t("Insert before")}
             labelClass="col-md-3"
-            divClass="col-md-8">
-            <option
-              key={"predecessorLabelEmpty"}/>
-            {props.environments && props.environments.map(env =>
-              <option
-                key={env.label}
-                value={env.label}>
-                {env.name}
-              </option>
-            )}
-          </Select>
+            divClass="col-md-8"
+            isClearable
+            options={props.environments}
+            getOptionValue={option => option != null ? option.label : null}
```

Comment:
I suspect nullable options would be a common use case (correct me if I'm wrong here?), in which case it could be useful to move this into the select component and handle it centrally there.

Reply:
I'll have a look at that then, thanks for the suggestion

Reply:
done